sbnobj/Common/Calibration/TrackCaloSkimmerObj.h

```diff
@@ -190,163 +112,97 @@ namespace sbn {
-
-    std::vector<int> daughter_pdg; //!< Pandora PDG codes of daughter PFP's
-    std::vector<unsigned> daughter_nsp; //!< Number of space points in each daughter
+    std::vector<WireInfo> wires0;    //!< List of wire information on plane 0
+    std::vector<WireInfo> wires1;    //!< List of wire information on plane 1
+    std::vector<WireInfo> wires2;    //!< List of wire information on plane 2
+
+    float t0PFP            = std::numeric_limits<float>::lowest(); //!< Particle-Flow-Particle (Pandora) T0. Derived from cathode crossing
+    float t0CRTTrack       = std::numeric_limits<float>::lowest(); //!< t0 from CRT Track (SBND)
+    float t0CRTHit         = std::numeric_limits<float>::lowest(); //!< t0 from CRT Hit (ICARUS)
+    float t0CRTSpacePoint  = std::numeric_limits<float>::lowest(); //!< t0 from CRT SpacePoint (SBND)
+    float crtMatchingScore = std::numeric_limits<float>::lowest(); //<! An assessment of the quality of the match made for whicht0 == 1,2,3
+    int whicht0     = -1;           //!< Which T0 producer was used to tag. 0 is Pandora, 1 is CRTTrack 2 is CRTHit, 3 is CRTSpacePoint
+    float xShiftCRT = 0.f;          //<! If whicht0 == 1,2,3 the amount by which the track was shifted in x
+    int id          = -1;           //!< ID of track
+    int cryostat    = -1;           //!< Cryostat number of track
+    bool clear_cosmic_muon = false; //!< Whether Pandora thinks the track is "clearly" a cosmic
+    Vector3D start;                 //!< Start position of track [cm]
+    Vector3D end;                   //!< End position of track [cm]
+    Vector3D dir;                   //!< Direction of track 
+    Vector3D PCAdir;                //!< Track Direction as fitted from PCA 
+    float length = -1.;             //!< Length of track [cm]
+
+    float hit_min_time_p0_tpcE = float(-100000.f); //!< Min hit time of track on plane 0 TPC E
+    float hit_max_time_p0_tpcE = float(-100000.f); //!< Max hit time of track on plane 0 TPC E
+    float hit_min_time_p1_tpcE = float(-100000.f); //!< Min hit time of track on plane 1 TPC E
+    float hit_max_time_p1_tpcE = float(-100000.f); //!< Max hit time of track on plane 1 TPC E
+    float hit_min_time_p2_tpcE = float(-100000.f); //!< Min hit time of track on plane 2 TPC E
+    float hit_max_time_p2_tpcE = float(-100000.f); //!< Max hit time of track on plane 2 TPC E
+    float hit_min_time_p0_tpcW = float(-100000.f); //!< Min hit time of track on plane 0 TPC W
+    float hit_max_time_p0_tpcW = float(-100000.f); //!< Max hit time of track on plane 0 TPC W
+    float hit_min_time_p1_tpcW = float(-100000.f); //!< Min hit time of track on plane 1 TPC W
+    float hit_max_time_p1_tpcW = float(-100000.f); //!< Max hit time of track on plane 1 TPC W
+    float hit_min_time_p2_tpcW = float(-100000.f); //!< Min hit time of track on plane 2 TPC W
+    float hit_max_time_p2_tpcW = float(-100000.f); //!< Max hit time of track on plane 2 TPC W
```

Comment:
You don't need any conversion here: `-100000.f` is explicitly already a `float`.
```suggestion
    float hit_min_time_p0_tpcE = -100000.f; //!< Min hit time of track on plane 0 TPC E
    float hit_max_time_p0_tpcE = -100000.f; //!< Max hit time of track on plane 0 TPC E
    float hit_min_time_p1_tpcE = -100000.f; //!< Min hit time of track on plane 1 TPC E
    float hit_max_time_p1_tpcE = -100000.f; //!< Max hit time of track on plane 1 TPC E
    float hit_min_time_p2_tpcE = -100000.f; //!< Min hit time of track on plane 2 TPC E
    float hit_max_time_p2_tpcE = -100000.f; //!< Max hit time of track on plane 2 TPC E
    float hit_min_time_p0_tpcW = -100000.f; //!< Min hit time of track on plane 0 TPC W
    float hit_max_time_p0_tpcW = -100000.f; //!< Max hit time of track on plane 0 TPC W
    float hit_min_time_p1_tpcW = -100000.f; //!< Min hit time of track on plane 1 TPC W
    float hit_max_time_p1_tpcW = -100000.f; //!< Max hit time of track on plane 1 TPC W
    float hit_min_time_p2_tpcW = -100000.f; //!< Min hit time of track on plane 2 TPC W
    float hit_max_time_p2_tpcW = -100000.f; //!< Max hit time of track on plane 2 TPC W
```